src/index.css

```
@@ -58,12 +61,12 @@ h2 {
 
 h1 {
   font-size: 1.625rem;
-  line-height: 2,375rem;
+  line-height: 2, 375rem;
 }
 
 h2 {
   font-size: 1.375rem;
-  line-height: 2,0625rem;
+  line-height: 2, 0625rem;
```

just delete the extra space
```suggestion
  line-height: 2,0625rem;
```